src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -69,9 +85,9 @@ export const ReservesGuardDescription = ({ formData }) => {
   );
 };
 
-export const title10DatesRequired = (formData) => get('view:isTitle10Activated', formData, false);
+export const title10DatesRequired = (formData: FormDataType): boolean => get('view:isTitle10Activated', formData, false);
 
-export const isInFuture = (errors, fieldData) => {
+export const isInFuture = (errors: { addError: (string) => mixed }, fieldData: string): void => {
```

Comment:
So this is an interesting thing. I was confused about how to define a function type for a bit, but after I read the documentation on it, it makes a ton of sense.